src/components/graph/graph.config.js

```diff
@@ -66,6 +66,8 @@
  *
  * <img src="https://github.com/danielcaldas/react-d3-graph/blob/master/docs/rd3g-zoom-animation.gif?raw=true" width="820" height="480"/>
  *
+ * @param {boolean} [freezeAllDragEvents=false] - <a id="freeze-all-drag-events" href="#freeze-all-drag-events">🔗</a> Disables manipulation of graph through drag
+ * and drop. This includes dragging graph elements, panning and zooming. *Note: this property can only be set in the first mount, it does not update dynamically.*
```

Comment:
> *Note: this property can only be set in the first mount; it does not update dynamically.*

Can I confirm whether you've tested out if this statement is true for the new property?

Reply:
Yes I have. I'll confirm again, but it doesn't seem to take on edits in the sandbox, for instance.

Reply:
Confirmed.